### PR TITLE
Updated project for Xcode 9

### PR DESCRIPTION
**Pull request by nrbrook**

### CODE REVIEW

File: Library/SRCommon.h [hunk 1]
```diff
@@ -78,7 +78,7 @@ FOUNDATION_STATIC_INLINE UInt32 SRCocoaToCarbonFlags(NSEventModifierFlags aCocoa
 
     @discussion Throws NSInternalInconsistencyException if bundle cannot be found.
 */
-NSBundle *SRBundle();
+NSBundle *SRBundle(void);
```
Comment (Kentzo) [inline Library/SRCommon.h]: @nrbrook Whom issued the warning and what was the message?

**Comment by Kentzo** [comment]
Thanks!

**Comment by nrbrook** [comment]
https://stackoverflow.com/questions/44473146/this-function-declaration-is-not-a-prototype-warning-in-xcode-9

**Comment by Kentzo** [comment]
Years of C++ totally made me forgot this one :)

**Pull request merged**